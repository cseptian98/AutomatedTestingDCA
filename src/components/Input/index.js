import React, {useState} from 'react';
import {View, TextInput, TouchableOpacity, StyleSheet} from 'react-native';
import axiosConfig from '../../api/BaseConfig';

const InputTodo = () => {
  const [title, setTitle] = useState('');

  const doCreate = () => {
    const onSuccess = ({data}) => {
      console.log('debug success', data);
    };

    const onFailure = error => {
      console.log('debug error', error.response.data);
    };

    axiosConfig.post('api/TodoLists', {title}).then(onSuccess).catch(onFailure);
  }

  return (
    <View style={styles.container}>
      <View style={styles.inputView}>
        <TextInput
          style={styles.textInput}
          placeholder="Add Todo List..."
          placeholderTextColor="#000"
          onChangeText={title => setTitle(title)}
        />
      </View>
      <TouchableOpacity style={styles.button} onPress={doCreate}>
        {/* ICON */}
      </TouchableOpacity>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    flexDirection: 'row',
    marginTop: 20,
  },
  inputView: {
    borderRadius: 10,
    flexDirection: 'row',
  },
  textInput: {
    width: 300,
    padding: 10,
    marginLeft: 10,
    marginRight: 10,
    marginBottom: 20,
    borderRadius: 10,
    backgroundColor: '#FFD500',
  },
  button: {
    width: 50,
    marginBottom: 20,
    borderRadius: 50,
    backgroundColor: '#FFD500',
  },
});

export default InputTodo;
