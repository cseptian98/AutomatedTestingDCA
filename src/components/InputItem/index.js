import React, {useState} from 'react';
import {View, TextInput, TouchableOpacity} from 'react-native';
import Icon from 'react-native-vector-icons/MaterialIcons';
import axiosConfig from 'api/BaseConfig';
import styles from './InputItems.styles';

const InputItem = () => {
  const listId = 1;
  const [title, setTitle] = useState('');

  const createNewItem = () => {
    const onSuccess = ({data}) => {
      console.log('debug success', data);
      setTitle('');
    };

    const onFailure = error => {
      console.log('debug error', error);
    };

    axiosConfig
      .post('api/TodoItems', {listId, title})
      .then(onSuccess)
      .catch(onFailure);
  };

  return (
    <View style={styles.container}>
      <View style={styles.inputView}>
        <TextInput
          style={styles.textInput}
          placeholder="Add Todo Item..."
          placeholderTextColor="#000"
          onChangeText={title => setTitle(title)}
        />
      </View>
      <TouchableOpacity style={styles.button} onPress={createNewItem}>
        <Icon name="add" size={24} />
      </TouchableOpacity>
    </View>
  );
};

export default InputItem;
