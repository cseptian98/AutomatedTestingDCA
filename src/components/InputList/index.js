import React, {useState} from 'react'
import {View, TextInput, TouchableOpacity, Image} from 'react-native'
import Icon from 'react-native-vector-icons/MaterialIcons'
import styles from './InputList.styles'
import axiosConfig from 'api/BaseConfig'

const InputList = ({createList}) => {
  const [title, setTitle] = useState('')

  // const createNewList = () => {
  //   const onSuccess = ({data}) => {
  //     console.log('debug success', data)
  //     setTitle('')
  //   }

  //   const onFailure = error => {
  //     console.log('debug error', error)
  //   }

  //   axiosConfig.post('api/TodoLists', {title}).then(onSuccess).catch(onFailure)
  // }

  return (
    <View style={styles.container}>
      <View>
        <Image
          style={styles.image}
          source={require('assets/images/list.png')}
        />
      </View>
      <View style={styles.inputView}>
        <TextInput
          style={styles.textInput}
          placeholder="Add Todo List..."
          placeholderTextColor="#FFF"
          onChangeText={title => setTitle(title)}
        />
        <TouchableOpacity
          style={styles.button}
          onPress={() => createList(title)}>
          <Icon name="add" size={30} color="#FFF" />
        </TouchableOpacity>
      </View>
    </View>
  )
}

export default InputList
