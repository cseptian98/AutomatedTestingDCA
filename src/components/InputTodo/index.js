import React from 'react';
import {View, TextInput, TouchableOpacity, StyleSheet} from 'react-native';

const InputTodo = () => {
  return (
    <View style={styles.container}>
      <View style={styles.inputView}>
        <TextInput
          style={styles.textInput}
          placeholder="Add Todo List..."
          placeholderTextColor="#000"
        />
      </View>
      <TouchableOpacity style={styles.button}>
        {/* ICON */}
      </TouchableOpacity>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    flexDirection: 'row',
  },
  inputView: {
    borderRadius: 10,
    flexDirection: 'row',
  },
  textInput: {
    width: 300,
    padding: 10,
    marginLeft: 10,
    marginRight: 10,
    marginBottom: 20,
    borderRadius: 10,
    backgroundColor: '#FFD500',
  },
  button: {
    width: 50,
    marginBottom: 20,
    borderRadius: 50,
    backgroundColor: '#FFD500',
  },
});

export default InputTodo;
