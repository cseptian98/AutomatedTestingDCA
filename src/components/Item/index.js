import React from 'react'
import {TouchableOpacity, Text} from 'react-native'
import styles from './Item.styles'

const Item = ({title, onDelete}) => (
  <TouchableOpacity style={styles.item} onPress={onDelete}>
    <Text style={styles.title}>{title}</Text>
  </TouchableOpacity>
)

export default Item
