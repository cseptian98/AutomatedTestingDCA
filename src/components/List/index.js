import React from 'react';
import {TouchableOpacity, Text, View} from 'react-native';
import styles from './List.styles';

const List = ({title, onDelete}) => (
  <TouchableOpacity style={styles.item} onPress={onDelete}>
    <Text style={styles.title}>{title}</Text>
  </TouchableOpacity>
);

export default List;
