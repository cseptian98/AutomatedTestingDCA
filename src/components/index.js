import FormUpdateItem from './TodoItems/FormUpdateItem/FormUpdateItem'
import TodoLists from './TodoLists/TodoLists'
import TodoItems from './TodoItems/TodoItems'
import User from './User/User'

export {FormUpdateItem, TodoLists, TodoItems, User}