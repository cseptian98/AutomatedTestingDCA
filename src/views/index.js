import HomeScreen from 'views/Home/Home'
import LoginScreen from 'views/Login/Login'
import RegisterScreen from 'views/Register/Register'

export {HomeScreen, LoginScreen, RegisterScreen}