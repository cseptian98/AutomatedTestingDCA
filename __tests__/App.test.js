import {LoginScreen, RegisterScreen} from 'views'
import {FormUpdateItem, TodoLists, TodoItems, User} from 'components'
import renderer from 'react-test-renderer'
import React from 'react'
import {waitFor} from '@testing-library/react-native'

describe('Testing', () => {
  describe('Login Screen', () => {
    it('renders correctly', async () => {
      await waitFor(() => {
        const tree = renderer.create(<LoginScreen />).toJSON()
        expect(tree).toMatchSnapshot()
      })
    })
  })
  describe('Register Screen', () => {
    const params = {url: 'api/Auth/register'}
    it('renders correctly', async () => {
      await waitFor(() => {
        const screen = renderer
          .create(<RegisterScreen route={{params}} />)
          .toJSON()
        expect(screen).toMatchSnapshot()
      })
    })
  })
  describe('TodoLists Screen', () => {
    const params = {url: 'api/TodoLists'}
    it('renders correctly', async () => {
      await waitFor(() => {
        const screen = renderer.create(<TodoLists route={{params}} />).toJSON()
        expect(screen).toMatchSnapshot()
      })
    })
  })
  describe('TodoItems Screen', () => {
    const params = {url: 'api/TodoItems'}
    it('renders correctly', async () => {
      await waitFor(() => {
        const screen = renderer.create(<TodoItems route={{params}} />).toJSON()
        expect(screen).toMatchSnapshot()
      })
    })
  })
  describe('Form Update Item Screen', () => {
    const params = {url: 'api/TodoItems/UpdateItemDetails'}
    it('renders correctly', async () => {
      await waitFor(() => {
        const screen = renderer
          .create(<FormUpdateItem route={{params}} />)
          .toJSON()
        expect(screen).toMatchSnapshot()
      })
    })
  })
  describe('User Screen', () => {
    const params = {url: 'api/User'}
    it('renders correctly', async () => {
      await waitFor(() => {
        const screen = renderer.create(<User route={{params}} />).toJSON()
        expect(screen).toMatchSnapshot()
      })
    })
  })
})
