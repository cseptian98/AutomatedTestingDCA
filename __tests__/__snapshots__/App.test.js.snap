// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Form Update Item Screen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFF",
      "flex": 1,
      "justifyContent": "flex-start",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/images/update_item.png",
      }
    }
    style={
      Object {
        "height": 200,
        "marginBottom": 16,
        "marginTop": 24,
        "width": 280,
      }
    }
    testID="TEST_ID_IMAGE_UPDATE_ITEM"
  />
  <View
    style={
      Object {
        "borderBottomColor": "#212121",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "margin": 10,
        "width": "90%",
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Note"
      placeholderTextColor="#000"
      rejectResponderTermination={true}
      style={
        Object {
          "flex": 1,
        }
      }
      testID="TEST_ID_NOTES_ITEM"
      underlineColorAndroid="transparent"
      value=""
    />
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#0D47A1",
        "borderRadius": 16,
        "height": 50,
        "justifyContent": "center",
        "marginTop": 20,
        "opacity": 1,
        "width": "90%",
      }
    }
    testID="TEST_ID_BUTTON_UPDATE_DETAIL"
  >
    <Text
      style={
        Object {
          "color": "#FFF",
          "fontSize": 16,
        }
      }
    >
      Update Item
    </Text>
  </View>
</View>
`;

exports[`Testing Login Screen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFF",
      "flex": 1,
      "justifyContent": "flex-start",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/images/login.png",
      }
    }
    style={
      Object {
        "height": 200,
        "marginBottom": 16,
        "marginTop": 24,
        "width": 280,
      }
    }
    testID="TEST_ID_IMAGE_LOGIN"
  />
  <View
    style={
      Object {
        "borderBottomColor": "#212121",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "margin": 10,
        "width": "90%",
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      keyboardType="email-address"
      onChangeText={[Function]}
      placeholder="Email"
      placeholderTextColor="#000"
      rejectResponderTermination={true}
      style={
        Object {
          "flex": 1,
        }
      }
      testID="TEST_ID_EMAIL_LOGIN"
      underlineColorAndroid="transparent"
      value=""
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#212121",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "margin": 10,
        "width": "90%",
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Password"
      placeholderTextColor="#000"
      rejectResponderTermination={true}
      secureTextEntry={true}
      style={
        Object {
          "flex": 1,
        }
      }
      testID="TEST_ID_PASSWORD_LOGIN"
      underlineColorAndroid="transparent"
      value=""
    />
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#000",
              "fontSize": 24,
            },
            undefined,
            Object {
              "fontFamily": "Material Design Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        󰈉
      </Text>
    </View>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#0D47A1",
        "borderRadius": 16,
        "height": 50,
        "justifyContent": "center",
        "marginTop": 20,
        "opacity": 1,
        "width": "90%",
      }
    }
    testID="TEST_ID_BUTTON_LOGIN"
  >
    <Text
      style={
        Object {
          "color": "#FFF",
          "fontSize": 16,
        }
      }
    >
      Login
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#E3F2FD",
        "borderRadius": 16,
        "height": 50,
        "justifyContent": "center",
        "marginTop": 20,
        "opacity": 1,
        "width": "90%",
      }
    }
    testID="TEST_ID_CREATE_ACCOUNT"
  >
    <Text
      style={
        Object {
          "color": "#212121",
          "fontSize": 16,
        }
      }
    >
      Create Account
    </Text>
  </View>
</View>
`;

exports[`Testing Register Screen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFF",
      "flex": 1,
      "justifyContent": "flex-start",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/images/register.png",
      }
    }
    style={
      Object {
        "height": 200,
        "marginBottom": 16,
        "marginTop": 24,
        "width": 280,
      }
    }
    testID="TEST_ID_IMAGE_REGISTER"
  />
  <View
    style={
      Object {
        "borderBottomColor": "#212121",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "margin": 10,
        "width": "90%",
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      keyboardType="email-address"
      onChangeText={[Function]}
      placeholder="Email"
      placeholderTextColor="#000"
      rejectResponderTermination={true}
      style={
        Object {
          "flex": 1,
        }
      }
      testID="TEST_ID_EMAIL_REGISTER"
      underlineColorAndroid="transparent"
      value=""
    />
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#212121",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "margin": 10,
        "width": "90%",
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Password"
      placeholderTextColor="#000"
      rejectResponderTermination={true}
      secureTextEntry={true}
      style={
        Object {
          "flex": 1,
        }
      }
      testID="TEST_ID_PASSWORD_REGISTER"
      underlineColorAndroid="transparent"
      value=""
    />
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#000",
              "fontSize": 24,
            },
            undefined,
            Object {
              "fontFamily": "Material Design Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        󰈉
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#212121",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "margin": 10,
        "width": "90%",
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Confirmation Password"
      placeholderTextColor="#000"
      rejectResponderTermination={true}
      secureTextEntry={true}
      style={
        Object {
          "flex": 1,
        }
      }
      testID="TEST_ID_CONFIRMATION_PASSWORD"
      underlineColorAndroid="transparent"
      value=""
    />
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "#000",
              "fontSize": 24,
            },
            undefined,
            Object {
              "fontFamily": "Material Design Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        󰈉
      </Text>
    </View>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#0D47A1",
        "borderRadius": 16,
        "height": 50,
        "justifyContent": "center",
        "marginTop": 20,
        "opacity": 1,
        "width": "90%",
      }
    }
    testID="TEST_ID_BUTTON_REGISTER"
  >
    <Text
      style={
        Object {
          "color": "#FFF",
          "fontSize": 16,
        }
      }
    >
      Register
    </Text>
  </View>
</View>
`;

exports[`Testing TodoItems Screen renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#FFF",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "marginTop": 10,
      }
    }
  >
    <View>
      <Image
        source={
          Object {
            "testUri": "../../../src/assets/images/item.png",
          }
        }
        style={
          Object {
            "alignItems": "center",
            "height": 120,
            "justifyContent": "center",
            "width": 160,
          }
        }
        testID="TEST_ID_IMAGE_TODOITEM"
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderRadius": 10,
          "flexDirection": "column",
          "justifyContent": "center",
        }
      }
    >
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Add Todo Item..."
        placeholderTextColor="#FFF"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#0D47A1",
            "borderRadius": 10,
            "color": "#FFF",
            "marginBottom": 10,
            "marginLeft": 16,
            "padding": 10,
            "width": 200,
          }
        }
        testID="TEST_ID_TEXT_INPUT_ITEM"
        underlineColorAndroid="transparent"
        value=""
      />
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#0D47A1",
            "borderRadius": 10,
            "justifyContent": "center",
            "marginBottom": 10,
            "marginLeft": 16,
            "opacity": 1,
            "width": 200,
          }
        }
        testID="TEST_ID_BUTTON_SUBMIT_ITEM"
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "#FFF",
                "fontSize": 30,
              },
              undefined,
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
      </View>
    </View>
  </View>
  <RCTScrollView>
    <View />
  </RCTScrollView>
</RCTSafeAreaView>
`;

exports[`Testing TodoLists Screen renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#FFF",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "marginTop": 10,
      }
    }
  >
    <View>
      <Image
        source={
          Object {
            "testUri": "../../../src/assets/images/list.png",
          }
        }
        style={
          Object {
            "alignItems": "center",
            "height": 120,
            "justifyContent": "center",
            "width": 160,
          }
        }
        testID="TEST_ID_IMAGE_TODOLIST"
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "borderRadius": 10,
          "flexDirection": "column",
          "justifyContent": "center",
        }
      }
    >
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Add Todo List..."
        placeholderTextColor="#FFF"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#0D47A1",
            "borderRadius": 10,
            "color": "#FFF",
            "marginBottom": 10,
            "marginLeft": 16,
            "padding": 10,
            "width": 200,
          }
        }
        testID="TEST_ID_TEXT_INPUT_LIST"
        underlineColorAndroid="transparent"
        value=""
      />
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#0D47A1",
            "borderRadius": 10,
            "justifyContent": "center",
            "marginBottom": 10,
            "marginLeft": 16,
            "opacity": 1,
            "width": 200,
          }
        }
        testID="TEST_ID_BUTTON_SUBMIT_LIST"
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "#FFF",
                "fontSize": 30,
              },
              undefined,
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
      </View>
    </View>
  </View>
  <RCTScrollView>
    <View />
  </RCTScrollView>
</RCTSafeAreaView>
`;

exports[`Testing User Screen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/images/profil.png",
      }
    }
    style={
      Object {
        "height": 200,
        "marginBottom": 30,
        "marginTop": 30,
        "width": 240,
      }
    }
    testID="TEST_ID_IMAGE_USER"
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#B71C1C",
        "borderRadius": 25,
        "color": "#000",
        "height": 50,
        "justifyContent": "center",
        "marginTop": 20,
        "opacity": 1,
        "width": "80%",
      }
    }
    testID="TEST_ID_BUTTON_LOGOUT"
  >
    <Text
      style={
        Object {
          "color": "#FFF",
        }
      }
    >
      Logout
    </Text>
  </View>
</View>
`;
